homeassistant/components/poolsense/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Connect to the device",
+        "data": {
+          "host": "Host"
```

Comment:
```suggestion
          "host": "[%key:common::config_flow::data::host%]"
```

Reply:
it also looks like you may be missing keys here for email, password, and serial

Reply:
I have now added these in. Thank you.